M2/Macaulay2/packages/gfanInterface.m2

```diff
@@ -4436,6 +4436,15 @@ doc///
 	 assert equalMPL(B,Bprime)
 	 ///
 
+-- TEST gfanConvertToNewRing x NCAlgebra (cf. #3600)
+	TEST ///
+	needsPackage "gfanInterface";
```

Comment:
Thanks for adding this test!  This first line (loading `gfanInterface`) isn't necessary since a package is always loaded before running its tests.

Reply:
Whoops, sorry.. thank you!